### PR TITLE
PMM-2607 Auto adding footer to PMM demo

### PR DESCRIPTION
**Pull request by adivinho**

### CODE REVIEW

**Comment by rnovikovP** [comment]
@adivinho  can yo check codacy.com comment. can we do something with it?

**Comment by adivinho** [comment]
Fixed

File: misc/cleanup-dash.py [hunk 1]
```diff
@@ -121,20 +121,19 @@ def set_hide_controls(dashboard):
...
+                id_ += 1
 
-                print('ID #%s was not unique - changed to #%s' % (
-                    panel['id'], id_
-                ))
-                dashboard['rows'][row_index]['panels'][panel_index]['id'] = id_
+            print('ID #%s was not unique - changed to #%s' % (
+                panel['id'], id_
+            ))
+            dashboard['panels'][panel_index]['id'] = id_
```
Comment (askomorokhov) [inline misc/cleanup-dash.py]: looks like an indentation has been lost here. now the loop does nothing except incrementation.

File: misc/cleanup-dash.py [hunk 2]
```diff
@@ -144,6 +143,25 @@ def drop_some_internal_elements(dashboard):
             del dashboard['__inputs']
         if '__requires' in element:
             del dashboard['__requires']
+        if 'panels' in element:
+            for panel_index, panel in enumerate(dashboard['panels']):
+                if 'datasource' in panel:
+                    if panel['datasource'] == "${DS_PROMETHEUS}":
```
Comment (askomorokhov) [inline misc/cleanup-dash.py]: This is correct technically, but some conventions does not encourage mix single and double quotes for simple strings.

File: misc/cleanup-dash.py [hunk 3]
```diff
@@ -194,14 +212,57 @@ def add_annotation(dashboard):
...
...a>\n  </div>\n</center>\n<hr>\n<link rel=\"stylesheet\" type=\"text/css\" href=\"//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.css\" />\n<script src=\"//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.js\"></script>\n<script>\nfunction bbb(){\n  \n  setTimeout(function (){ \n  window.cookieconsent.initialise({\n    \"palette\": {\n      \"popup\": {\n        \"background\": \"#eb6c44\",\n        \"text\": \"#ffffff\"\n      },\n      \"button\": {\n        \"background\": \"#f5d948\"\n      }\n    },\n    \"theme\": \"classic\",\n    \"content\": {\n      \"message\": \"This site uses cookies and other tracking technologies to assist with navigation, analyze your use of our products and services, assist with promotional and marketing efforts, allow you to give feedback, and provide content from third parties. If you do not want to accept cookies, adjust your browser settings to deny cookies or exit this site.\",\n      \"dismiss\": \"Allow cookies\",\n      \"link\": \"Cookie Policy\",\n      \"href\": \"https://www.percona.com/cookie-policy\"\n    }\n  })},3000)};\n  \n  \n  window.addEventListener(\"load\",bbb());\n\n\n\n</script>",
```
Comment (askomorokhov) [inline misc/cleanup-dash.py]: Please, use single quotes here, and no escapes of double quotes in html  :)

File: misc/cleanup-dash.py (hunk 2 above)
Comment (adivinho) [reply misc/cleanup-dash.py]: Thx, quotes have been replaced.

File: misc/cleanup-dash.py (hunk 3 above)
Comment (adivinho) [reply misc/cleanup-dash.py]: Thx, quotes have been replaced.

**Pull request closed without merging**